packages/gestalt/src/SheetMobile/FullPage.js

```diff
@@ -0,0 +1,270 @@
+    };
+  }, [onDismiss]);
+
+  const updateShadows = useCallback(() => {
+    const target = contentRef.current;
+    if (!target) {
+      return;
+    }
+
+    const hasVerticalScrollbar = target.clientHeight < target.scrollHeight;
+    setShowTopShadow(hasVerticalScrollbar && target.scrollTop > 0);
+    setShowBottomShadow(
+      hasVerticalScrollbar && target.offsetHeight + target.scrollTop < target.scrollHeight,
+    );
+  }, []);
+
+  useEffect(() => {
+    window.addEventListener('resize', updateShadows);
+    return () => {
+      window.removeEventListener('resize', updateShadows);
+    };
+  }, [updateShadows]);
+
+  useEffect(() => {
+    updateShadows();
+  }, [updateShadows]);
+
+  const dismissButtonRef = useRef();
+
+  useEffect(() => {
+    if (dismissButtonRef.current) {
+      dismissButtonRef.current.focus();
+    }
+  }, [dismissButtonRef]);
+
+  const id = useId();
+
+  useEffect(() => {
+    // When SheetMobile is full page displayed in mobile browser, the body scroll is still accessible. Here we disable to just allow the scrolling within Modal
+    if (window && window.body?.style?.overflow) {
+      window.body.style.overflow = 'hidden';
+    }
+    return () => {
+      if (window && window.body?.style?.overflow) {
+        window.body.style.overflow = 'auto';
+      }
+    };
+  }, []);
+
+  return (
+    <StopScrollBehavior>
+      <TrapFocusBehavior>
+        <div
+          id={id}
+          aria-label={accessibilityLabel ?? defaultAccessibilityLabel}
+          className={classnames(sheetMobileStyles.container, sheetMobileStyles.fullPageContainer)}
+          role={role}
+        >
+          <Backdrop closeOnOutsideClick={false}>
+            <div
+              className={classnames(sheetMobileStyles.fullPageWrapper, focusStyles.hideOutline)}
+              tabIndex={-1}
+              style={{ width: '100%' }}
+            >
+              <Box flex="grow" position="relative" display="flex" direction="column" width="100%">
+                <Box
+                  padding={4}
+                  borderStyle={showTopShadow ? 'raisedTopShadow' : undefined}
+                  position="relative"
+                  fit
+                >
+                  <Flex justifyContent="center" alignItems="center" gap={4}>
+                    {backIconButton ? (
+                      <Flex.Item flex="none">
+                        <IconButton
+                          accessibilityLabel={backIconButton.accessibilityLabel}
+                          icon="arrow-back"
+                          iconColor="darkGray"
+                          size="lg"
+                          onClick={({ event }) =>
+                            backIconButton?.onClick({ event, onDismissStart: onExternalDismiss })
+                          }
+                        />
+                      </Flex.Item>
+                    ) : null}
+                    {!backIconButton && showDismissButton ? (
```

Comment:
@dangerismycat  It would be dismissed tapping outside. I snapshotted this yesterday from the presentation. 

![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/10593890/229167538-4812b371-173b-4b28-bce3-2ae9fb46280b.png)


There isn't always a dismiss button